PostHog/PostHogSDK.swift

```diff
@@ -245,6 +252,22 @@ let maxRetryDelay = 30.0
         }
         return props
     }
+    
+    private func getSessionId() -> String? {
+        storage?.getString(forKey: .sessionId)
+    }
+    
+    private func setSessionId(_ newValue: String) {
+        storage?.setString(forKey: .sessionId, contents: newValue)
+    }
+    
+    private func getSessionLastTimestamp() -> TimeInterval? {
+        storage?.getDouble(forKey: .sessionLastTimestamp)
+    }
+    
+    private func setSessionLastTimestamp(_ newValue: TimeInterval) {
+        storage?.setDouble(forKey: .sessionLastTimestamp, contents: newValue)
+    }
```

Comment:
This can be all in-memory, if the app is closed/killed by the OS, it's a brand new session.

For reference https://github.com/PostHog/posthog-android/blob/main/posthog-android/src/main/java/com/posthog/android/internal/PostHogLifecycleObserverIntegration.kt

Reply:
Ok great, that simplifies things :)